source/tests/yats-test/util_test.cpp

```diff
@@ -5,46 +5,29 @@
 #include <yats/Util.h>
 
 
-TEST(is_unique_ptr_test, util_test)
+TEST(util_test, is_unique_ptr_test)
 {
 	ASSERT_TRUE(is_unique_ptr_v<std::unique_ptr<int>>);
```

Comment:
General question: Do we want to use `ASSERT_*` or `EXPECT_*`? I prefer `EXPECT_*` because multiple things can be checked in one test and only things that are required for the rest to pass need to be asserted. The fist assert fail will end the test, that does not happen on expect.